src/Analyzers/Analyzers/test/StartupAnalyzerTest.cs

```diff
@@ -4,347 +4,527 @@
 using System.Collections.Concurrent;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Analyzer.Testing;
 using Microsoft.CodeAnalysis;
 using Xunit;
+using VerifyCS = Microsoft.AspNetCore.Analyzers.CSharpCodeFixVerifier<
+    Microsoft.AspNetCore.Analyzers.StartupAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 
 namespace Microsoft.AspNetCore.Analyzers
 {
-    public class StartupAnalyzerTest : AnalyzerTestBase
+    public class StartupAnalyzerTest
     {
-        public StartupAnalyzerTest()
+        [Fact]
+        public async Task StartupAnalyzer_FindsStartupMethods_StartupSignatures_Standard()
+        {
+            var source = @"// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Analyzers.TestFiles.StartupAnalyzerTest
+{
+    public class StartupSignatures_Standard
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app)
         {
-            StartupAnalyzer = new StartupAnalyzer();
 
-            Runner = new AnalyzersDiagnosticAnalyzerRunner(StartupAnalyzer);
+        }
+    }
+}
+";
+
+            await VerifyCS.VerifyCodeFixAsync(source, source);
```

Comment:
📝 This is the recommended form for analyzer tests, even without a code fix. In this form, the test verifies that no code fix is presented for the scenario. If in the future a code fix is implemented the test will start to fail, and can easily be updated to show the impact of the code fix when run on the input.